src/embed.ts

```diff
@@ -467,9 +467,18 @@ export abstract class Embed {
    */
   private setIframe(isLoad: boolean): void {
     if(!this.iframe) {
-      const iframeHtml = `<iframe style="width:100%;height:100%;" src="${this.config.embedUrl}" scrolling="no" allowfullscreen="true"></iframe>`;
-      this.element.innerHTML = iframeHtml;
-      this.iframe = <HTMLIFrameElement>this.element.childNodes[0];
+      var iframeContent = document.createElement("IFRAME");
```

Comment:
If I understood the question correctly, yes. the iframes are differentiated by the containers' id.